tests/expected/dead-invalid-access-via-raw/main.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
```

Comment:
Please add a comment that explains what this test is checking. It would be useful to add a reference to #3061 as well.